src/compute/src/communication.rs

```diff
@@ -82,11 +82,15 @@ pub async fn initialize_networking(
     }
 
     match initialize_networking_from_sockets(sockets, *process, *workers, Box::new(|_| None)) {
-        Ok((stuff, guard)) => Ok((
-            stuff.into_iter().map(GenericBuilder::ZeroCopy).collect(),
-            Box::new(guard),
-        )),
+        Ok((stuff, guard)) => {
+            info!(process = process, "successfully initialized network");
+            Ok((
+                stuff.into_iter().map(GenericBuilder::ZeroCopy).collect(),
+                Box::new(guard),
+            ))
+        }
         Err(err) => {
+            warn!(process = process, "failed to initialize network: {err}");
```

Comment:
```suggestion
            warn!(process, "failed to initialize network: {err}");
```
I *think* this works as well? But just a nit in any case.